openwisp_firmware_upgrader/admin.py

```diff
@@ -76,7 +76,7 @@ def upgrade_selected(self, request, queryset):
             # returning None will display the change list page again
             return None
         upgrade_all = request.POST.get('upgrade_all')
-        upgrade_related = request.POST.get('upgrade_related')
+        upgrade_related = request.POST.get('upgrade_related')  # UNUSED!!
```

Comment:
is this not being used at all there?

Reply:
I don't see how it's used:

This is the logic simplified:

```
upgrade_all = request.POST.get('upgrade_all')
upgrade_related = request.POST.get('upgrade_related') 
if upgrade_all or upgrade_related:
    batch = build.batch_upgrade(firmwareless=upgrade_all)
```

Should I remove it?

Reply:
I've opened an issue to track it -> https://github.com/openwisp/openwisp-firmware-upgrader/issues/81